bindata/network/network-metrics/001-daemonset.yaml

```diff
@@ -38,7 +38,7 @@ spec:
                 operator: DoesNotExist
               - key: network.operator.openshift.io/dpu
                 operator: DoesNotExist
-      priorityClassName: "system-node-critical"
+      priorityClassName: "openshift-user-critical"
```

Comment:
This is definitely the right one for metrics, seeing it's user facing and doesn't have an impact on core node functionality. Good one!